tests/tls-issuerdn/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
```

Comment:
I recently learned that when we want to use a pcap from a pre-existing sv test, we can also add:
```yaml
pcap: ../existing-test-name/pcap-file.pcap
```
:)

Reply:
As seen in: https://github.com/OISF/suricata-verify/blob/2eca390bdd825fd7c709a18ef0884146c3cb4c7e/tests/detect-udp-flow-rule-01/test.yaml

Reply:
so you mean I dont need to give the pcap at all and reference it here ?

Reply:
Yes it works. thanks